client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -54,6 +69,39 @@ export default connectOptions(
 					<ThanksModal
 						site={ site }
 						source={ 'list' } />
+					{ config.isEnabled( 'manage/themes/upload' ) &&
+						<ThemeUploadCard
+							label={ translate( 'WordPress.com themes' ) }
+						/>
+					}
+					<ThemesSelection
```

Comment:
E.g. we'll want to provide a `jetpackActivate` option instead of a regular `activate`. But that's something we should address in subsequent iterations, now that I think of it -- not yet in this initial PR. We'll be able to parallelize some of that work.